gettsim/parameters/soz_vers_beitr.yaml

```
@@ -10,13 +10,26 @@ beitr_satz:
                     sonderbeitrag - bezahlt von Arbeitnehmern, ab Jul 2005 bis 2014
                     ermäßigt - ermäßigter Satz (§243 SGB V)
                     zusatz - durchschnittlicher Zusatzbeitragssatz.
+      ges_pflegev: Beitragssätze für gesetzliche Pflegeversicherung.
+                    standard - Beitragssatz für Arbeitnehmer
+                    zusatz_kinderlos - Beitragszuschlag für kinderlose Versicherte,
+                    die das 23. Lebensjahr vollendet haben. Wird vom Arbeitnehmer
+                    alleine getragen. Eingeführt 2005.
+      arbeitsl_v:  Beitragssätze zur Arbeitslosenversicherung für Arbeitnehmer.
+      ges_rentenv: Beitragssätze zur allgemeinen Rentenversicherung für Arbeitnehmer.
     en:
-      ges_krankenv: Statury Contribution Rates for mandatory health insurance.
+     ges_krankenv: Statury Contribution Rates for mandatory health insurance.
```

```suggestion
      ges_krankenv: Statury Contribution Rates for mandatory health insurance.
```
Here we need an additional whitespace (two more as in the intendation level before)